### PR TITLE
Fixes for tests/bugs introduced in vaccel

### PR DESCRIPTION
**Pull request by papazof**

### CODE REVIEW

File: .github/workflows/utils.yml [hunk 1]
```diff
@@ -192,15 +193,15 @@ jobs:
...
           make
-          make test ARGS="-V"
+          CTEST_OUTPUT_ON_FAILURE=1 make test ARGS="-V"
           cd ../
         shell: bash
 
       - name: Calculate coverage
         run: |
-          gcovr --cobertura=./docs/coverage/coverage_pr.xml --gcov-executable gcov-12 -s -e "build/_deps/*"
+          gcovr --filter src/ --filter plugins/ --cobertura=./docs/coverage/coverage_pr.xml --gcov-executable gcov-12 -s -e "build/_deps/*"
```
Comment (jlcsf) [inline .github/workflows/utils.yml:204]: I think we can get rid of the  ```-e "build/_deps/*"``` here if we include the ```--filter```

Comment (papazof) [reply .github/workflows/utils.yml:204]: yep, makes sense

**Review by ananos — APPROVED** [review]
thanks @papazof, lgtm!

**Comment by ananos** [comment]
ignore the coverage failure as it depends on the main branch

**Pull request merged**